recipe/conda_forge_ci_setup/upload_or_check_non_existence.py

```diff
@@ -135,7 +138,11 @@ def upload_or_check(recipe_dir, owner, channel, variant):
             for path in new_distributions:
                 upload(token_fn, path, owner, channel)
                 print('Uploaded {}'.format(path))
-        return True
+
+        if _should_validate():
+            return request_copy([b[2] for b in built_distributions], channel)
```

Comment:
Can you this inside the `with` statement? Some feedstocks have issues with deleting the temp file above.

Reply:
Changed. Note that the copy request doesn't need the binstar token in the tempfile made by the context so I don't think this should matter.